### PR TITLE
Feature/message attributes

### PR DESCRIPTION
**Pull request by heliocentrist**
Added the possibility to set MessageAttributeNames option when reading messages from the queue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "sqs-duplex",
-  "version": "0.1.0",
+  "name": "@icemobile/sqs-duplex",
```
Comment (brianc) [inline package.json:2]: you're changing the package name here?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+max_line_length = 80
```
Comment (brianc) [inline .editorconfig:8]: I don't think this file really goes in source control - it's specific to your editor I believe.

**Comment by brianc** [comment]
I appreciate the PR! I haven't used this module in a long time.  Happy to turn over maintenance and ownership to you...but in this PR you actually changed the module name in the package.json file - why did you do that?

**Comment by heliocentrist** [comment]
Hi Brian, sorry, this PR was created by mistake, it was intended for the forked repo, that's why I've closed it. Please let me know if you'd like me to make the same PR to your original repo, I'll be happy to do that.

**Comment by brianc** [comment]
Oh my bad! I didn't notice it was closed! :grin: ignore me. :)

**Pull request closed without merging**